cli_tools/common/utils/storage/object_handle.go

```diff
@@ -0,0 +1,74 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package storage
+
+import (
+	"context"
+	"io"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/domain"
+)
+
+// ObjectHandleCreator is responsible for creating GCS Object Handle
+type ObjectHandleCreator struct {
+	ctx context.Context
+	sc  *storage.Client
+}
+
+// CreateObjectHandle creates GCS Object Handle
+func (ohc *ObjectHandleCreator) CreateObjectHandle(
```

Comment:
ref to bucket-relatednaming style, we can call it "GetObject". "CreateObjectHandle" is a bit confusion for callers